modules/completion/init.zsh

```diff
@@ -40,6 +40,7 @@ if [[ $_comp_path(#qNmh-20) ]]; then
 else
   mkdir -p "$_comp_path:h"
   compinit -i -d "$_comp_path"
+  touch "$_comp_path"
```

Comment:
Could you add a clarifying comment in the code  of _why_ we need this? Nothing fancy, but it's helpful when you see things that seem useless in teh code to have a quick comment explaining why it's not useless...